corehq/apps/fixtures/views.py

```diff
@@ -541,13 +541,17 @@ def _get_fixture_upload_args_from_request(request, domain):
             replace = True
         elif replace.lower() == "false":
             replace = False
-        is_async = request.POST["async"]
-        is_async = is_async.lower() == "true"
     except Exception:
         raise FixtureAPIRequestError(
             "Invalid post request."
             "Submit the form with field 'file-to-upload' and POST parameter 'replace'")
 
+    try:
+        is_async = request.POST.get("async")
+        is_async = is_async.lower() == "true"
+    except Exception:
+        is_async = False
```

Comment:
I think you could change this to

```
is_async = request.POST.get("async", "").lower() == "true"
```

I think that's clearer and safer than catching all possible exceptions.

Reply:
ok done : )